__tests__/app.test.js

```diff
@@ -112,4 +112,48 @@ describe("app ", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id/comments", () => {
+    test("respond with a status of 200, and display an array of comments for the given review_id", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((commentsByReviewId) => {
+          expect(Array.isArray(commentsByReviewId.body)).toBe(true);
+          expect(commentsByReviewId.body).toHaveLength(3);
+        });
+    });
+    test("each comment must have comment_id,votes,created_at,author,body and review_id properties", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((commentsByReviewId) => {
+          commentsByReviewId.body.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
+            expect(comment).toHaveProperty("votes");
+            expect(comment).toHaveProperty("created_at");
+            expect(comment).toHaveProperty("author");
+            expect(comment).toHaveProperty("body");
+            expect(comment).toHaveProperty("review_id");
+          });
+        });
+    });
```

Comment:
I'd advise checking the values as well as the keys here using either 
expect(comment).toHaveProperty("comment_id", expect.any(Number))
OR 
users.forEach((user) => {
          expect(user).toEqual(
            expect.objectContaining({
              username: expect.any(String)...etc